### ISSUE TITLE
Can mup deploy an app using child_process to run shell?

### ISSUE THREAD

**Issue by langker** [arunoda/meteor-up-legacy#255]
I use mup to deploy my meteor app on my sever,but I can't run shell code in my app which does well in my local computer,can you test it?

**Comment by davidrinnan** [comment]
can it have something to do with user privileges
as far as I know MUP steps down from root, only root in order to get access to port < 1000

it states 

> > stepping down to gid: meteoruser
> > stepping down to uid: meteoruser

in mup logs

**Comment by arunoda** [comment]
It can spawn child_processes. May be your shell script is not bundled with
meteor. Meteor does not bundle files other than js/css/html into the final
version. That might be the issue.

**Comment by langker** [comment]
``` bash
if (Meteor.isClient) {
  // counter starts at 0
  Session.setDefault('counter', 0);

  Template.hello.helpers({
    counter: function () {
      return Session.get('counter');
    }
  });

  Template.hello.events({
    'click button': function () {
      Meteor.call("setBlogList");
    }
  });
}

if (Meteor.isServer) {
  Meteor.startup(function () {
    Meteor.methods({
    "setBlogList":function(){
        var run = Npm.require("child_process");
        run.exec("mkdir ~/memeda");
    }
    });
  });
}
```

Thank you for replay my question,I use meteor defalut demo making a simple app using shell,it runs well in my local computer,but after deploy it ,it failed,I just don't know how to debug it on server.Can you test it  as well?Thank you

**Comment by langker** [comment]
Sorry,it's my bug,thank you for helping !

**Comment by langker** [comment]
close #255

**Comment by langker** [comment]
Closes #255
